### PR TITLE
Remove .DS_Store files from a Git repository

### PR DESCRIPTION
**Pull request by patsevanton**

### CODE REVIEW

**Comment by arey** [comment]
Good check. Thanks @patsevanton Could you add this file to the `.gitignore` please?

**Comment by patsevanton** [comment]
Add .DS_Store to the .gitignore

File: .gitignore [hunk 1]
```diff
@@ -17,4 +17,8 @@ target/
 # Branch switching
 generated/
 .DS_Store
```
Comment (arey) [inline .gitignore:19]: I'm sorry, the `.DS_Store` file was already declared. I don't think we have to declare full paths.

**Comment by arey** [comment]
Thank you @patsevanton

**Pull request merged**